packages/object-has/index.tests.ts

```diff
@@ -0,0 +1,16 @@
+import has from './index';
+
+// OK
+
+has({a: 1}, 'a');
+has({a: {b: 1}}, 'a.b');
+has(undefined, 'a');
+const sym = Symbol('foo');
+has({[sym]: 1}, sym);
```

Comment:
can you add a test with array of strings as second arg

Reply:
Added